src/runtime_src/core/pcie/driver/linux/xocl/xocl_xgq.c

```diff
@@ -20,12 +20,18 @@
 
 #define CLIENT_ID_BITS 7
 #define MAX_CLIENTS (1 << CLIENT_ID_BITS)
+#define CLIENT_ID_MASK (MAX_CLIENTS - 1)
 
 static uint16_t xocl_xgq_cid;
```

Comment:
Uninitalized variable.

Reply:
Global variables are automatically initialized to 0 at the time of declaration. This is define in the C standard.